libbpfgo/helpers/example_tracelisten_test.go

```diff
@@ -1,15 +1,13 @@
-package helpers_test
+package helpers
 
 import (
 	"fmt"
 	"os"
-
-	"github.com/aquasecurity/tracee/libbpfgo/helpers"
 )
 
 func ExampleTracePipeListen_usage() {
```

Comment:
Now that you changed this file package to "helpers" do we want to expose this function as part of the helpers package?

Reply:
This was a mistake, reverting to `helpers_test` package.